packages/sbom-tools/test/helpers.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+import { rimrafSync } from 'rimraf';
+
+export const importFixture = async (file: string) =>
+  (await import(`./fixtures/${file}`)).default;
+
+export const toCleanup: string[] = [];
+
+export function setupTempDir(structure: Record<string, string>): string {
+  const tempDirPath = path.join(os.tmpdir(), `sbom-tools-${Date.now()}`);
+
+  fs.mkdirSync(tempDirPath, { recursive: true });
+  toCleanup.push(tempDirPath);
+
+  for (const [entryPath, content] of Object.entries(structure)) {
+    const absoluteEntryPath = path.join(tempDirPath, ...entryPath.split('/'));
```

Comment:
```suggestion
    const absoluteEntryPath = path.join(tempDirPath, entryPath);
```

Reply:
Is that ok in win? I'm always adding entries like this: `{'node_modules/my-file.js': '....'`}

Reply:
Yes, `/` is also a path separator on Windows (basically aliases to `\`), and Node.js also does understand this properly:

```js
> path.win32.join('a', 'b/c/d/')
'a\\b\\c\\d\\'
> path.posix.join('a', 'b/c/d/')
'a/b/c/d/'
```

Reply:
awesome, thought that the separator was always different, thanks!